client/my-sites/importer/error-pane.jsx

```diff
@@ -79,7 +79,7 @@ class ImporterError extends PureComponent {
 		}
 
 		return this.props.translate(
-			'%(errorDescription)s{{br/}}Make sure you are using a valid WordPress export file in XML or ZIP format. {{cs}}Still need help{{/cs}}?',
```

Comment:
Can we use the `this.props.importerEngine` and have more specific error messages for different engine types. 

if ( this.props.importerEngine === 'substack' ) {
					return this.props.translate( '%(errorDescription)s{{br/}}Make sure you are using a valid ZIP export file downloaded from substack.com. {{cs}}Still need help{{/cs}}?', ...
}

Reply:
We have it already, look at:
https://github.com/Automattic/wp-calypso/blob/d54be4cb294a8a06e76b1a8e6db0450165341091/client/my-sites/importer/error-pane.jsx#L70-L92

The problem is for imports other than from WordPress we still mention WordPress in the error message. My change just removes the "WordPress" word to make it more generic.

Reply:
Sure, we can adjust those error messages later but for now, I think it's good enough as it is.

Reply:
Sounds good!